src/lib/utils/generateAvatar.ts

```diff
@@ -0,0 +1,24 @@
+import { HF_TOKEN, TEXT_TO_IMAGE_MODEL } from "$env/static/private";
+import { generateFromDefaultEndpoint } from "$lib/server/generateFromDefaultEndpoint";
+import { HfInference } from "@huggingface/inference";
+
+export async function generateAvatar(description?: string, name?: string): Promise<File> {
+	const queryPrompt = `Generate a prompt for an image-generation model for the following: 
+Name: ${name}
+Description: ${description}
+`;
+	const imagePrompt = await generateFromDefaultEndpoint({
```

Comment:
is the small model being used here or not?

Reply:
Yes indeed, like for websearch & title summarization